sdk/core/core-auth/review/core-auth.api.md

```diff
@@ -9,9 +9,15 @@ import { AbortSignalLike } from '@azure/abort-controller';
 // @public
 export interface AccessToken {
     expiresOnTimestamp: number;
+    refreshOn?: number;
```

Comment:
should this be `refreshOnTimestamp` to mirror the above? I'm assuming it is ticks since the epoch?